pylint/checkers/similar.py

```diff
@@ -18,8 +18,10 @@
 """
 
 from __future__ import print_function
+import ast
```

Comment:
Let's replace this with `astroid` instead. The API is similar with the `ast` module so you shouldn't need to modify anything other than the import.

Reply:
Done.